Exiled.API/Features/Player.cs

```diff
@@ -782,6 +789,11 @@ public Item CurrentItem
             }
         }
 
+        /// <summary>
+        /// Gets the armor that the player is currently wearing. Value will be <see langword="null"/> if the player is not wearing any armor.
+        /// </summary>
+        public Armor CurrentArmor => Inventory.TryGetBodyArmor(out BodyArmor armor) ? (Armor)Item.Get(armor) : null;
```

Comment:
Should we move this to the HumanRole class?

Reply:
I don't think so. The rest of the inventory stuff is in the player class, armor should be as well